Resources/Private/Partials/Backend/SharedApplicationActions/Show.html

```diff
@@ -3,8 +3,12 @@
 
 <div class="row">
 	<div class="col-md-6 summary-block table-container">
-		<h3><f:translate id="be.label.ApplicationSingle.Overview.PersonalData"></f:translate></h3>
-
+		<h3>
+			<f:translate id="be.label.ApplicationSingle.Overview.PersonalData"></f:translate>
+			<f:link.action action="show" target="_blank" arguments="{application: application, print: 1}" class="print">
+				<img src="{f:uri.resource(path: 'Icons/btn_print.png')}" alt="print" />
```

Comment:
Please use `f:translate` for `alt` here. And replace the ` />` with `>`, we only use HTML5. ;-)